contracts/modules/dispute/DisputeModule.sol

```diff
@@ -222,7 +221,11 @@ contract DisputeModule is
     /// @param disputeId The dispute id
     /// @param decision The decision of the dispute
     /// @param data The data to set the dispute judgement
-    function setDisputeJudgement(uint256 disputeId, bool decision, bytes calldata data) external nonReentrant {
+    function setDisputeJudgement(
```

Comment:
given we are pausing `setDisputeJudgment` - we could pause `tagDerivativeIfParentInfringed` as both are forms of applying tags.

Reply:
Do you think we should pause dispute judgements?

Reply:
a95ca7e